test/jquery.js

```diff
@@ -2,7 +2,7 @@
 ( function() {
 	/* global loadTests: false */
 
-	var path = window.location.pathname.split( "test" )[ 0 ],
+	var path = window.location.pathname.substring( 0, window.location.pathname.lastIndexOf( "test" ) ),
```

Comment:
@gibson042 addressed in https://github.com/jquery/jquery/pull/3757/commits/29c2d1258fd68ee804ed4cc5f02e6fcb2916f404. Thanks!